package.json

```diff
@@ -28,53 +31,60 @@
     "lib/"
   ],
   "scripts": {
-    "delete:node_modules": "rm -rf node_modules",
-    "delete:package-lock": "rm -f package-lock.json",
+    "check:environment": "bun run --silent check:environment:symlinks && bun run --silent check:environment:versions",
+    "check:environment:symlinks": "check-environment-symlinks",
+    "check:environment:versions": "check-environment-versions",
+    "check:formatting": "prettier --check --no-editorconfig .",
+    "check:lint-conflicts": "eslint-config-prettier ./lib/eslintBaseConfig.json",
+    "delete:node-modules": "rm -rf node_modules",
+    "delete:package-lock": "rm -f bun.lockb",
     "format:code": "prettier --write --no-editorconfig .",
+    "install:bun:expected-version": "bun --print '(await import(Bun.env.npm_package_json)).packageManager.split(\"@\")[1]' | (read BUN_VERSION; if [ $BUN_VERSION == `bun --version` ]; then echo Bun version $BUN_VERSION is already installed; else echo Installing Bun version $BUN_VERSION... && curl -fsSL https://bun.sh/install | bash -s \"bun-v$BUN_VERSION\"; fi)",
     "list:eslint:disable-directives": "rg '/(/|\\*+)[ \\t]*eslint-disable[^*]*(\\*+/)?'",
-    "list:todo-comments": "rg --only-matching '(TODO|FIXME):[a-zA-Z0-9\\t .,;?]+'",
-    "reinstall": "npm run --silent delete:node_modules && npm run --silent delete:package-lock && npm i",
-    "reinstall:use-lock-file": "npm run --silent delete:node_modules && npm ci",
-    "test": "eslint --max-warnings 0 --report-unused-disable-directives test/**/*.{js,jsx,mjs,ts,tsx}",
-    "validate:formatting": "prettier --check --no-editorconfig .",
-    "validate:linting:eslint": "eslint-config-prettier ./lib/eslintBaseConfig.json"
+    "list:todo-comments": "rg --only-matching '(TODO|FIXME)([\t ]+\\[[^\\]]+\\])?:[\\w\\s!\"#$%&'\\''()+,./:;<=>?@\\^_`{|}~-]+'",
+    "postinstall": "bun run --silent check:environment:versions",
```

Comment:
I think you didn't intend this, but `postinstall` scripts are also ran on consumers' machine, so with those changes, if one tries to run `npm i @babbel/eslint-config`, after installing the package, npm will attempt to run this script.

Screenshot on my machine after publishing to Verdaccio:
![Screenshot 2024-07-24 at 16 05 24](https://github.com/user-attachments/assets/540ec6b6-2f29-4f5f-8768-b55451549bb4)

Reply:
Oh no, I forgot about that. Thank you for pointing that out!

Reply:
I tried a few ways to keep it in, but I ended up removing it

Reply:
Could you have forgotten to push? I don't seem to see any changes

Reply:
I pushed to `origin` (my fork) not `upstream`. I just pushed now. Thanks for pointing that out.